src/components/Datepicker/Datepicker.vue

```diff
@@ -149,16 +149,22 @@ export default {
       return this.focusedDate.getFullYear();
     },
     prevMonthDisabled () {
-      return this.min && this.min.getMonth() === this.focusedMonth && this.min.getFullYear() === this.focusedYear;
+      return this.minDate && this.minDate.getMonth() === this.focusedMonth && this.minDate.getFullYear() === this.focusedYear;
     },
     nextMonthDisabled () {
-      return this.max && this.max.getMonth() === this.focusedMonth && this.max.getFullYear() === this.focusedYear;
+      return this.maxDate && this.maxDate.getMonth() === this.focusedMonth && this.maxDate.getFullYear() === this.focusedYear;
+    },
+    minDate () {
+      return this.min || new Date(new Date().setMonth(new Date().getMonth() - 12));
+    },
+    maxDate () {
+      return this.max || new Date(new Date().setMonth(new Date().getMonth() + 12));
```

Comment:
I added these in place of the `default` logic on `min` and `max`.  For some reason when mocking `Date` it was not able to override when used in a `default` statement but it worked fine anywhere else.